jenkins/image_building/dib_elements/centos-ci/post-install.d/01_install

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Why is this script in `post-install.d`? Shouldn't it be `install.d`?

Reply:
We run package install in ci-base element, that is mostly installs the packages and run in install phase. 
Since we do some more OS specific configuration changes, i decided to keep it in post install phase. It is not much different, since both run in chroot.

Reply:
Sorry I still don't get how that's related? Imo if it's a script to install stuff, then we should have it in `install.d`.
For e.g. you could name it `02_os_specific` or something like that, we can make sure it always runs after the `01_` script on the ci-base element.

Reply:
Maybe a stupid question, can we split it and have both install.d and post-install.d? So we would put 01 in install.d and 02 in post-install.d. That would make the most sense to me.

Reply:
yes, we can. I will do it like this then.